mlibrary/prejudice
# Removing favorites.

Removing favorites.
Adding docker / docker-compose.

<details open>
<summary> :hand:&nbsp;&nbsp;<strong>Wait!</strong></summary>

Let's make sure your pull request is ready. Ask yourself...

:white_check_mark:&nbsp;&nbsp;"Did I do a good job testing my own code?"
:white_check_mark:&nbsp;&nbsp;"Is my code easy to read and understand?"
:white_check_mark:&nbsp;&nbsp;"Have I written unit tests? Do the unit tests fully cover my code?"
:white_check_mark:&nbsp;&nbsp;"Have I tested the repository's behavior to make sure my code didn't break anything?"
:negative_squared_cross_mark:&nbsp;&nbsp;"Did I include unrelated work that could be in its own pull request?"

Once you are ready, be sure to delete this `details` section before opening your pull request.
</details>

# Overview
We've retired the Favorites back-end, and have removed Favorites from the Search UI.  This PR removes the code to interact with favorites.

> This pull request resolves/closes/fixes [LIBSEARCH-715](https://mlit.atlassian.net/browse/LIBSEARCH-715).

## Anything else?
_Optional_. If appropriate, provide additional notes the reviewers should be made aware of. They can be about any "gotchas" or even changes that are expected to be made in the future, but are not applied to this pull request.

## Testing
List instructions on how to test the pull request. Some examples:

- Run the tests to make sure they pass (`npm run test`).
  - Break the new/updated unit tests to make sure they're working properly.
- Build the repository (`npm run build`).
- Make sure the PR is consistent in repositories that use `prejudice`.
- _How to use the feature_.
